src/test/java/stepdefinition/SuccessfulSubscriptionStep.java

```diff
@@ -0,0 +1,37 @@
+package stepdefinition;
+
+import com.giantshoulder.pageobject.SuccessfulSubscriptionPage;
+import cucumber.api.java.en.Then;
+
+/**
+ * @author Gabriella_Havadtoi
+ */
+public class SuccessfulSubscriptionStep extends BaseStepDefinition {
+
+    SuccessfulSubscriptionPage successfulSubscriptionPage = new SuccessfulSubscriptionPage(driver);
+
+    @Then("I should get a successful subscription message for learn faster")
+    public void i_should_get_a_successful_subscription_message_for_learn_faster() {
+        String message = " Your email is confirmed and the FREE download will arrive in your inbox shortly.";
```

Comment:
This could come from the feature file.

Reply:
Fixed (maybe) in 7e10b9665d692268601505913831d793d73aef7e!